src/main/java/core/basesyntax/service/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Inventory;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport(Inventory inventory) {
```

Comment:
The method `getReport` should not directly depend on the `Inventory` class. It's better to pass the data needed for generating the report as a parameter to the method. This will make the method more reusable and easier to test.